.github/workflows/docfx-gh-pages.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy DocFX with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["feature/docfx"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Github Pages
+        uses: actions/configure-pages@v3
+        
+      - name: Build DocFX Site
+        uses: VakuWare/docfx-pdf-action@v1.4.0
```

Comment:
Should we just use the original docfx action? https://github.com/marketplace/actions/docfx-action this one is a copy of that one but with additional PDF support that we don't need

Reply:
I had trouble with the original